client/src/components/forms/form-wrapper.component.test.tsx

```diff
@@ -1,20 +1,25 @@
 import { describe, expect, it } from 'vitest';
 import { FormWrapper } from './form-wrapper.component';
 import { render, screen } from '~/test/utils';
-import { LoginForm } from '~/accounts/authentication/login-form.component';
-
-let loginUser: () => void;
 
 describe('form-wrapper', () => {
-  beforeEach(() => {
-    loginUser = vi.fn();
-  });
-  it('should render form-wrapper on the page with login form', () => {
+  it('should render form-wrapper on the page with a form', () => {
     render(
       <FormWrapper>
-        <LoginForm loginUser={loginUser} />
+        <form aria-label="Add your details">
+          <label htmlFor="firstName">First Name:</label>
+          <input type="text" id="firstName" />
+          <label htmlFor="lastName">Last Name:</label>
+          <input type="text" id="lastName" />
+
+          <button type="submit">Submit Me</button>
+        </form>
       </FormWrapper>,
     );
     expect(screen.getByRole('article')).toBeInTheDocument();
+    expect(screen.getByRole('form')).toBeInTheDocument();
+    expect(screen.getByRole('textbox', { name: 'First Name:' })).toBeInTheDocument();
+    expect(screen.getByRole('textbox', { name: 'Last Name:' })).toBeInTheDocument();
```

Comment:
When specifying a name, testing-library generally prefer regex, so I'd get in the habit of doing the same. This is perfectly fine, but most people use regex, so they can ignore things like capitalization. It also means you can use partial names as well, so you could do something like `/first.*/i`, that would find any `textbox` with the word `first|First|FIRST` etc, it makes your code more resilient to change. Other people like to be as explicit as possible, it's a hard one to call, but I tend to go with what the developers of the library themselves do, so I always use regex, where I can.